site/src/sphinx/server-cors-service.rst

```diff
@@ -0,0 +1,107 @@
+.. _CORSWiki: https://en.wikipedia.org/wiki/Cross-origin_resource_sharing
+.. _server-cors-service:
+
+Configure CORS service
+========================
+
+Armeria provides a way to configure Cross-origin resource sharing(CORS) policy for specified origins or
+any origin via :api:`CorsServiceBuilder`. For more information about CORS, visit CORSWiki_.
+
+
+Allowing any origin
+-------------------
+To configure CORS Service allowing any origin (*), use ``CorsServiceBuilder.forAnyOrigin()``
+
+.. code-block:: java
+
+    HttpService myService = new AbstractHttpService() {
+                ...
+    }
+    ServerBuilder sb = new ServerBuilder().service("/message", myService.decorate(
+            CorsServiceBuilder.forAnyOrigin()
+                              .allowCredential()
+                              .allowRequestMethods(HttpMethod.POST, HttpMethod.GET)
+                              .allowRequestHeaders(HttpHeaderNames.of("allow_request_header"))
+                              .exposeHeaders(HttpHeaderNames.of("expose_header_1"),
+                                             HttpHeaderNames.of("expose_header_2"))
+                              .preflightResponseHeader("x-preflight-cors", "Hello CORS")
+                              .newDecorator()));
+
+
+
+
+Allowing the specified origins
+------------------------------
+To configure CORS Service allowing the specified origins, use ``CorsServiceBuilder.forOrigins`` or
```

Comment:
- the specified origins -> specific origins ?
- `forOrigins` -> `forOrigins()`
- `forOrigin` -> `forOrigin()`